pkg/models/repository_configuration_test.go

```diff
@@ -59,6 +60,22 @@ func (suite *ModelsSuite) TestCreateInvalidVersion() {
 	assert.True(suite.T(), strings.Contains(res.Error.Error(), "version"))
 }
 
+func (suite *ModelsSuite) TestCreateDuplicateVersion() {
```

Comment:
Should there be a test for Updating too? I notice the whole file only tests create, so it's not the precedent, but the beforeupdate hook could theoretically differ from the beforecreate hook.

Reply:
added as part of this test